packaging/templates/puppynet/archive/config.toml

```diff
@@ -119,7 +119,7 @@ gcmode = "archive"
     # maxheaderhistory = 1024
     # maxblockhistory = 1024
     # maxprice = "5000000000000"
-    # ignoreprice = "2"
+    # ignoreprice = "30000000000"
 
 [telemetry]
     metrics = true
```

Comment:
prometheus-addr = ""  if we are leaving it blank so after updating node we need to manually add 0.0.0.0:7071..

Reply:
It was blank by default, If you think we are going to use in all nodes, Add in all 8 files, I have no issue.

Reply:
yes we need it in all nodes .. adding